internal/broker/bind_create_test.go

```diff
@@ -376,11 +378,33 @@ func TestCreateBindingEndpoint(t *testing.T) {
 		binding = unmarshal(t, response)
 		assert.Equal(t, firstInstanceSecondBinding, binding)
 		assert.Equal(t, firstInstanceSecondBindingDB.Kubeconfig, binding.Credentials.(map[string]interface{})["kubeconfig"])
-		assertClusterAccess(t, response, "secret-to-check-first", binding)
+		assertClusterAccess(t, "secret-to-check-first", binding)
+	})
+
+	t.Run("should delete created binding", func(t *testing.T) {
```

Comment:
Is there a test case for deleting already deleted resource (idempotency)?

Reply:
That is not the case for `DELETE` binding API endpoint. From the spec https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#response-9 we learn that: "
410 Gone | MUST be returned if the Service Binding does not exist."

Reply:
Agreed to be address in separate PR.